File: BMO2/BMO2/Atbash/frmAtbash.vb
Public Class frmAtbash
    Private Sub BtnCifrar_Click(sender As Object, e As EventArgs) Handles btnCifrar.Click
        Dim at As MetodoAtbash = New MetodoAtbash()
        txtEncriptadas.Items.Add(at.ZXY(txtPalabras.Text))
        txtResultados.Text = at.cambio
    End Sub

    Private Sub SalirToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles salirToolStripMenuItem.Click
        Application.Exit()
    End Sub

    Private Sub CriptogramasToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles criptogramasToolStripMenuItem.Click
        Dim Ce As New MetodomsnAtbash
        Ce.Abrir()
        Me.Close()
    End Sub
End Class

File: BMO/BMO/Atbash/frmAtbash.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace BMO.Atbash
{
    public partial class frmAtbash : Form 
    {
        
        public frmAtbash()
        {
            InitializeComponent();
        }

        private void BtnCifrar_Click(object sender, EventArgs e)
        {
            MetodoAtbash at = new MetodoAtbash();
            txtEncriptadas.Items.Add(at.ZXY(txtPalabras.Text));
            txtResultados.Text = at.cambio;
        }       

        private void SalirToolStripMenuItem_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }

        private void CriptogramasToolStripMenuItem_Click(object sender, EventArgs e)
        {
            MetodosmsnAtbash ob = new MetodosmsnAtbash();
            this.Close();
            ob.AbrirCesar();
        }
    }
}
